service/http/service.go

```diff
@@ -128,7 +129,10 @@ func New(ctx context.Context, serviceName string, config Config, clientApplicati
 		}))
 	}
 
-	httpServer := &http.Server{Handler: kitNetHttp.OpenTelemetryNewHandler(handler, serviceName, tracerProvider)}
+	httpServer := &http.Server{
+		Handler:     kitNetHttp.OpenTelemetryNewHandler(handler, serviceName, tracerProvider),
+		ReadTimeout: time.Second * 10,
```

Comment:
@jkralik  gosec complained about a security issues if ReadTimeout is not set, 10 sec seemed like a reasonable value to use, or do you think some other should be used?

	

> // ReadTimeout is the maximum duration for reading the entire
> 	// request, including the body. A zero or negative value means
> 	// there will be no timeout.
> 	//
> 	// Because ReadTimeout does not let Handlers make per-request
> 	// decisions on each request body's acceptable deadline or
> 	// upload rate, most users will prefer to use
> 	// ReadHeaderTimeout. It is valid to use them both.
> 	ReadTimeout time.Duration

Reply:
We need to extends http server configuration in hub about it - we need to load them from yaml file.

Reply:
https://github.com/plgd-dev/hub/pull/762